scripts/jointLklDM.C

```diff
@@ -579,8 +578,10 @@ void jointLklDM(TString configFileName="$GLIKESYS/rcfiles/jointLklDM.rc",Int_t s
 		hadcanvas[isample]->SetName(Form("hadcanvas_%d",isample));
 		hadcanvas[isample]->SetTitle(Form("IRFs and data for sample %d",isample));
 		hadcanvas[isample]->cd(5);
-		TLatex* ltchannel = new TLatex(0.8,0.8,strchannel);
-		ltchannel->SetTextSize(0.055);
+        TLatex* ltchannel;
+        if(nChannels == 1) ltchannel = new TLatex(0.8,0.8,strchannel);
+        else ltchannel = new TLatex(0.7,0.8,strchannel);
+        ltchannel->SetTextSize(0.055);
```

Comment:
Please indent

Reply:
The editor messed that up. Should be fixed with the next commit.